tgui/packages/tgui/interfaces/RIGSuit.js

```diff
@@ -0,0 +1,336 @@
+import { Fragment } from 'inferno';
+import { useBackend } from "../backend";
+import { Box, Button, Dropdown, LabeledList, ProgressBar, Section, Stack } from "../components";
+import { Window } from "../layouts";
+import { capitalize, toTitleCase } from 'common/string';
+
+export const RIGSuit = (props, context) => {
+  const { act, data } = useBackend(context);
+
+  const {
+    interfacelock,
+    malf,
+    aicontrol,
+    ai,
+  } = data;
+
+  let override = null;
+
+  if (interfacelock || malf) {
+    // Interface is offline, or a malf AI took over, either way, the user is
+    // no longer permitted to view this interface.
+    override = <Box color="bad">--HARDSUIT INTERFACE OFFLINE--</Box>;
+  } else if (!ai && aicontrol) {
+    // Non-AI trying to control the hardsuit while it's AI control overridden
+    override = <Box color="bad">-- HARDSUIT CONTROL OVERRIDDEN BY AI --</Box>;
+  }
+
+  return (
+    <Window
+      height={480}
+      width={550}
+      resizable>
+      <Window.Content scrollable>
+        {override || (
+          <Fragment>
+            <RIGSuitStatus />
+            <RIGSuitHardware />
+            <RIGSuitModules />
+          </Fragment>
+        )}
+      </Window.Content>
+    </Window>
+  );
+};
+
+const RIGSuitStatus = (props, context) => {
+  const { act, data } = useBackend(context);
+
+  const {
+    // Power Bar
+    chargestatus,
+    charge,
+    maxcharge,
+    // AI Control Toggle
+    aioverride,
+    // Suit Status
+    sealing,
+    sealed,
+    // Cover Locks
+    emagged,
+    securitycheck,
+    coverlock,
+    tracking_level,
```

Comment:
ideally move this tracking stuff to the vital monitor section

failing that, make them not appear if no vital monitor is installed

Reply:
Done